lib/matplotlib/tests/test_axes.py

```diff
@@ -260,6 +260,29 @@ def test_strmethodformatter_auto_formatter():
     assert ax.yaxis.get_minor_formatter().fmt == targ_strformatter.fmt
 
 
+def test_twinning():
+    # plot the axis
+    ax1 = plt.axes()
+
+    # get the currently position
+    pos1 = ax1.get_position()
+    # pos2 takes pos1 coords, and, alter the height by multiplying it by 0.6
+    pos2 = mtransforms.Bbox.from_extents([pos1.x0, pos1.y0,  pos1.width, pos1.height * 0.6])
+    
+    # twiny() ax1
+    ax1.twiny()
+
+    # set a new position
+    ax1.set_position(pos2)
+
+    # it gets the position, in a np.array, of ax1 after the twiny and the set_position
+    array1 = np.all(mtransforms.Bbox.get_points(ax1.get_position()))
```

Comment:
You actually don't need `np.all` here either. `np.all` will also return a boolean. The function `np.any` returns `True` if there are any (more than 0) non-zero elements in an array. `np.all` returns `True` if all of the elements of an array are nonzero.

To test if your test makes sense, try filling in something which you would expect to make the test fail. For example, you can change entirely how you calculate one of the arrays. In the example below, I've set `array2` to `np.all([True])`

```
# it gets the position, in a np.array, of ax1 after the twiny and the set_position
array1 = np.all(mtransforms.Bbox.get_points(ax1.get_position()))
# it gets the np.array of the point of pos2
array2 = np.all([True])
```

If you run this test, you will see that it passes. Btw, just in case you are not aware, if you are running a test locally with pytest, you can select only one test using the flag `-k`. 

```
pytest -v -k "test_twinning"
```

What you want is to assert that the output from `mtransforms.Bbox.get_points(ax1.get_position())` is equal to the output of `mtransforms.Bbox.get_points(pos2)`, correct? These will both return arrays so you should be able to use `np.array_equal` with those outputs as arguments.

Reply:
Okay, thanks for the help, I will see